src/Win32Tests/packages.config

```diff
@@ -1,5 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Castle.Core" version="4.4.0" targetFramework="net471" />
+  <package id="Moq" version="4.12.0" targetFramework="net471" />
```

Comment:
>12 [](start = 31, length = 2)

The rest of Axe.Windows is using Moq 4.11. Can we use that version here, as well? (Happy to update the version, but that feels like a separate PR)

Reply:
Updated